.env.example

```diff
@@ -1,3 +1,3 @@
 PINECONE_API_KEY=
 PINECONE_ENVIRONMENT=
-PINECONE_INDEX=
\ No newline at end of file
+PINECONE_INDEX=semantic-search
```

Comment:
I put a sample value for this into the template because when I was going through this project I was initially confused about the purpose of this field.